fuse_models/test/test_unicycle_2d_state_cost_function.cpp

```diff
@@ -128,7 +128,7 @@ TEST(CostFunction, evaluateCostFunction)
 
   for (size_t i = 0; i < num_parameter_blocks; ++i)
   {
-    EXPECT_MATRIX_NEAR(J_autodiff[i], J[i], std::numeric_limits<double>::epsilon())
+    EXPECT_MATRIX_NEAR(J_autodiff[i], J[i], 1e-15)
```

Comment:
Odd that this test should now fail.

Reply:
I'll double check that it passes before with epsilon, regardless std::numeric_limits<double>::epsilon() seems very small for a jacobian test. I'm assuming it doesn't pass now due to rotations in the residual computation

Reply:
That makes sense.